record-builder-test/src/test/java/io/soabase/recordbuilder/test/TestOptionsOnPackage.java

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
This is the wrong copyright. Please run `mvn license:format`

Reply:
Sorry, i do not agree.

the error is that it expect this license.
https://github.com/Randgalt/record-builder/blob/master/src/etc/header.txt

`Copyright 2019 Jordan Zimmerman`
- year is wrong
- `Jordan Zimmerman` do not write the class.

The license-checker should just test the license part.
```
Licensed under the Apache License, Version 2.0 (the "License");
....
```

Reply:
It's just a formality. If you can fix the license checker feel free. Otherwise we can't merge this.

Reply:
since you use an Apache license i would remove the maven plugin and add an GitHub action.
This helps Contrbutors because they can see this as an PR-Check.
https://github.com/eclipse/org.eclipse.sensinact.gateway/blob/future/prototype/.github/workflows/license.yml


would that be okay?

Reply:
No, I don't like the GHA. I like the Maven plugin because it's system independent. We should just take my name out and make it like this project: https://github.com/failsafe-lib/failsafe/blob/master/core/src/main/java/dev/failsafe/Bulkhead.java.

```
Copyright 2021 the original author or authors.
```

I can fix up the date with the plugin. I was planning on doing a cleanup/tech-debt PR anyway. I can do it there.

Reply:
yes, i thing fixing license and license-header-plugin is better your job

Reply:
now its `Copyright 2023 the original author or authors.`

Reply:
@stbischof please review https://github.com/Randgalt/record-builder/pull/150 if you can